organisms/organisation-unit-tree/new-features/select-all.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Select or deselect all available org units
+
+  Scenario: Select all available org units
+    Given an org unit tree with multiple selection enabled
+    When the user clicks 'Select all' control
+    Then all selectable org units are selected
+
+  Scenario: Deselect all available org units
+    Given an org unit tree with >1 selected org units
```

Comment:
`>1` could be `some`

Reply:
True. Especially as `≥1` is actually correct, `some` is easier!